### PR TITLE
Add attributes to loss_functions

### PR DESCRIPTION
**Pull request by maksymiuks**
Related to #246

### CODE REVIEW

File: R/misc_loss_functions.R [hunk 1]
```diff
@@ -27,21 +27,28 @@ loss_cross_entropy <- function(observed, predicted, p_min = 0.0001, na.rm = TRUE
...
+attr(loss_cross_entropy, "loss_name") <- "Cross-entropy"
+
+
+
 
 #' @rdname loss_functions
 #' @export
 loss_sum_of_squares <- function(observed, predicted, na.rm = TRUE)
   sum((observed - predicted)^2, na.rm = na.rm)
+attr(loss_sum_of_squares, "loss_name") <- "Sum of errors squares"
```
Comment (pbiecek) [inline R/misc_loss_functions.R:39]: Sum of squared residuals (SSR)

File: R/misc_loss_functions.R [hunk 2]
```diff
@@ -27,21 +27,28 @@ loss_cross_entropy <- function(observed, predicted, p_min = 0.0001, na.rm = TRUE
...
 
 #' @rdname loss_functions
 #' @export
 loss_root_mean_square <- function(observed, predicted, na.rm = TRUE)
   sqrt(mean((observed - predicted)^2, na.rm = na.rm))
+attr(loss_root_mean_square, "loss_name") <- "Root mean square error"
```
Comment (pbiecek) [inline R/misc_loss_functions.R:45]: Root mean square error (RMSE)

File: R/misc_loss_functions.R [hunk 3]
```diff
@@ -27,21 +27,28 @@ loss_cross_entropy <- function(observed, predicted, p_min = 0.0001, na.rm = TRUE
   p <- sapply(seq_along(observed), function(i)  predicted[i, observed[i]] )
   sum(-log(pmax(p, p_min)), na.rm = TRUE)
 }
+attr(loss_cross_entropy, "loss_name") <- "Cross-entropy"
```
Comment (pbiecek) [inline R/misc_loss_functions.R:30]: Cross entropy

File: R/misc_loss_functions.R (hunk 1 above)
Comment (maksymiuks) [reply R/misc_loss_functions.R]: Done

File: R/misc_loss_functions.R (hunk 3 above)
Comment (maksymiuks) [reply R/misc_loss_functions.R]: Done

File: R/misc_loss_functions.R (hunk 2 above)
Comment (maksymiuks) [reply R/misc_loss_functions.R]: Done

**Pull request merged**